src/templates/head_css.html

```diff
@@ -12,25 +12,32 @@
 {% if request.user.profile.dark_theme %}
 <link href="{% static 'css/bootstrap_slate.min.css' %}" rel="stylesheet">
 {% else %}
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
       integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
-      crossorigin="anonymous">
+      crossorigin="anonymous"> -->
+ <!-- Bootstrap JavaScript bundle (includes Popper) -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.3.2/js/bootstrap.bundle.min.js"></script>
```

Comment:
Why are you including this twice?  You already imported 5.0.2 in base.html.  Also, please consider best practices and if this can go at the bottom of the body.